docker/seccomp.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+def loadDefault( filename ):
+    with open( filename, 'r' ) as f:
+        return json.load( f )
+
+def inPlaceTransform( jdata ):
```

Comment:
As a side-note; I'd suggest sticking with python's default coding standard, PEP-8 which says method names should be `method_name` rather than `methodName`.

Reply:
PEP-8 does not apply to anything except system libraries.